Libraries/Opc.Ua.Gds.Server.Common/ApplicationsNodeManager.cs

```diff
@@ -304,7 +305,7 @@ protected async Task<ICertificateGroup> InitializeCertificateGroup(CertificateGr
             }
 
             ICertificateGroup certificateGroup = m_certificateGroupFactory.Create(
-                m_configuration.AuthoritiesStorePath, certificateGroupConfiguration);
+                m_GlobalDiscoveryServerConfiguration.AuthoritiesStorePath, certificateGroupConfiguration, m_configuration);
```

Comment:
could you change the class/constructor to not keep the whole ApplicationConfiguration? The app config can be reloaded and holding the whole copy I have some concerns unless its really necessary.
I believe there is only need to store the IssuerStorePath?